### DISCUSSION TITLE
Vulnerabilities discovered by docker desktop

### DISCUSSION THREAD

**Discussion by severfire** [FlowiseAI/Flowise#3429] [General]
Hi, I found that libs should be updated, there are many vulnerabilities found in packages used in FlowiseAI

![image](https://github.com/user-attachments/assets/d6a9cea4-5f8b-4db2-9a83-923614a90aad)

seems like this was not really talked about, even when someone showed the issue in the past:
https://github.com/FlowiseAI/Flowise/discussions/574

**Comment by HenryHengZJ** [comment]
@vincelwt have you came across with the lunary npm vulnerabilities?

**Comment by severfire** [comment]
the strange thing is on https://www.npmjs.com/package/lunary
we see version 0.7.13 as latest... and it shows two vulnerabilities in docker scout

and on https://github.com/lunary-ai/lunary
we see there is v1.5.8 available...

strange... or maybe I do not know something 

https://scout.docker.com/vulnerabilities/id/CVE-2024-5389?s=github&n=lunary&t=npm&vr=%3C1.4.9
https://scout.docker.com/vulnerabilities/id/CVE-2024-4146?s=github&n=lunary&t=npm&vr=%3C1.2.26
https://scout.docker.com/vulnerabilities/id/CVE-2024-5478?s=github&n=lunary&t=npm&vr=%3C%3D1.2.7
